src/NuGet.Server/Core/DefaultServiceResolver.cs

```diff
@@ -44,8 +53,12 @@ public object Resolve(Type type)
                 return _packageAuthenticationService;
             }
 
-
             return null;
         }
+
+        public void Dispose()
+        {
+            _packageRepository.Dispose();
+        }
```

Comment:
Suggestion: consider implementing the dispose pattern. https://msdn.microsoft.com/en-us/library/b1yfkh5e(v=vs.110).aspx#basic_pattern